src/App.tsx

```diff
@@ -1,14 +1,61 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todosStatus, setTodosStatus] = useState<boolean | null>(null);
+  const [todosQuery, setTodosQuery] = useState('');
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+
+  useEffect(() => {
+    getTodos().then(setTodos);
+  }, []);
+
+  const setterOfSelectedTodo = (pickedTodo: Todo | null) => {
+    setSelectedTodo(pickedTodo);
+  };
```

Comment:
why do you need this function if you can use `setSelectedTodo` right away?

Reply:
> why do you need this function if you can use `setSelectedTodo` right away?

my mistake, its bad naming of it, this is Lifting state up function, thats why i need to use it like this